test/domain/relation/_fixtures/relation.fixture.ts

```diff
@@ -47,5 +48,17 @@ await database.createRecord(RECORD_TYPE_RELATION, relation5);
 await database.createRecord(RECORD_TYPE_RELATION, relation6);
 await database.createRecord(RECORD_TYPE_RELATION, relation7);
 
-export { CREATOR0, CREATOR1, CREATOR2, CREATOR3, CREATOR4, CREATOR5, establish, explore, getFollowers, getFollowing, RELATION1, RELATION2, RelationAlreadyExists, RelationData, REQUESTER1, REQUESTER2, SortOptions };
+const QUERY0: RecordQuery = {
+    followerId: { EQUALS: REQUESTER2.id },
+    followingId: { EQUALS: CREATOR0 }
+};
+
+const QUERY1: RecordQuery = {
+    followerId: { EQUALS: johnDoe.id },
+    followingId: { EQUALS: CREATOR1 }
+};
+
+const UNKNOWN_REQUESTER = johnDoe;
+
+export { CREATOR0, CREATOR1, CREATOR2, CREATOR3, CREATOR4, CREATOR5, QUERY0, QUERY1, RECORD_TYPE_CREATOR, RECORD_TYPE_RELATION, RELATION1, RELATION2, REQUESTER1, REQUESTER2, RelationAlreadyExists, RelationData, SortOptions, UNKNOWN_REQUESTER, database, establish, explore, getFollowers, getFollowing };
```

Comment:
Can we break the export into logical groups to make them more readable?

```suggestion
export {
    CREATOR0, CREATOR1, CREATOR2, CREATOR3, CREATOR4, CREATOR5,
    QUERY0, QUERY1,
    RECORD_TYPE_CREATOR, RECORD_TYPE_RELATION,
    RELATION1, RELATION2,
    REQUESTER1, REQUESTER2,
    RelationAlreadyExists, RelationData, SortOptions, UNKNOWN_REQUESTER,
    database, establish, explore, getFollowers, getFollowing
};
```